backend/app/assets/javascripts/spree/backend/navigation.coffee

```diff
@@ -4,7 +4,18 @@ navHeight = ->
 checkSideBarFit = ->
   $('.admin-nav').toggleClass('fits', navHeight() < $(window).height())
 
-$ ->
-  $(".admin-nav-sticky, .admin-nav").stick_in_parent()
+checkSticky = ->
+  stickyElements = $('.admin-nav-sticky, .admin-nav')
+
+  if $(window).width() < 768
+    stickyElements.trigger 'sticky_kit:detach'
+  else
+    stickyElements.stick_in_parent(spacer: false)
```

Comment:
Latest Chrome, small desktop viewport (~1024) lower the screen height so the sticky feature get's triggered.